src/main/resources/templates/index.html

```diff
@@ -30,30 +30,33 @@ <h1 class="spacing-below-35 spacing-above-5" th:text="#{index.title}"></h1>
               class="button--small button"></a>
           </div>
         </div>
+        <img src="/assets/images/mother-reading-to-child.svg" alt="" class="width-one-half is-tablet-hidden">
       </div>
     </section>
     <section class="slab slab--white">
       <div class="grid">
-        <div class="index-child-care-image-container">
-            <img src="/assets/images/seal-of-illinois.png" alt="" style="width: 110px; height: 110px;">
-            <img src="/assets/images/IDHS-logo.png" alt="">
+        <div class="index-child-care-image-container width-one-fourth">
+            <img src="/assets/images/seal-of-illinois.png" alt="" style="width: 141px; height: 141px;">
+            <img src="/assets/images/IDHS-logo.png" alt="" style="width: 168px; height: auto">
+        </div>
+        <div class="width-one-half">
+          <h2 th:text="#{index.child-care.title}"></h2>
+          <p th:text="#{index.child-care.paragraph}"></p>
+          <a
+              target="_blank" rel="noopener noreferrer"
+              href="https://www.inccrra.org/about/sdasearch">
+            <span th:text="#{index.child-care.link}"></span>
+            <i class="icon-">open_in_new</i>
+          </a>
         </div>
-        <h2 th:text="#{index.child-care.title}"></h2>
-        <p th:text="#{index.child-care.paragraph}"></p>
-        <a
-            target="_blank" rel="noopener noreferrer"
-            href="https://www.inccrra.org/about/sdasearch">
-          <span th:text="#{index.child-care.link}"></span>
-          <i class="icon-">open_in_new</i>
-        </a>
       </div>
     </section>
     <section class="slab slab--light-purple">
       <div class="grid">
         <h2 th:text="#{general.eligibility-table.title}"></h2>
         <table class="eligibility-table">
           <thead>
-          <tr>
+          <tr class="color--idhs-base-blue bold">
             <th scope="col" th:text="#{general.eligibility-table.column.header-1}"></th>
             <th scope="col" th:text="#{general.eligibility-table.column.header-2}"></th>
```

Comment:
The `.bold` class has to be on a `<th>` to take effect, see it's [definition in custom.css for reference, aka `th.bold` means the class has to be on the element](https://github.com/codeforamerica/il-gcc-form-flow/blob/284e7c4e70a24b3e56f8b93333adbc5d99301d2c/src/main/resources/static/assets/css/custom.css#L146-L149).

Also looks like the design color changed from `black` to the `idhs-base-blue`. I like they way you did it better with the utility class. So in `custom.css` the `.bold` needs to be modified to not set the color:

```diff
/* in custom.css:146 */
.eligibility-table th.bold {
  font-weight: bold;
- color: var(--black);
}
```

```suggestion
          <tr>
            <th class="color--idhs-base-blue bold" scope="col" th:text="#{general.eligibility-table.column.header-1}"></th>
            <th class="color--idhs-base-blue bold" scope="col" th:text="#{general.eligibility-table.column.header-2}"></th>
```

Reply:
Why does the `bold` change the appearance of the font when the page is rendered?  I am going to change it but I'm still curious